<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewMilitaryDialog</class>
 <widget class="QDialog" name="AddNewMilitaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додавання Бійця</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbEnterSurname">
     <property name="locale">
      <locale language="Ukrainian" country="Ukraine"/>
     </property>
     <property name="text">
      <string>Введіть прізвище: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leSurname"/>
   </item>
   <item>
    <widget class="QLabel" name="lbEnterName">
     <property name="text">
      <string>Введіть ім'я:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item>
    <widget class="QLabel" name="lbEnterAge">
     <property name="text">
      <string>Введіть вік:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leAge"/>
   </item>
   <item>
    <widget class="QLabel" name="lEnterBloodType">
     <property name="text">
      <string>Виберіть гурпу крові:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbBloodType"/>
   </item>
   <item>
    <widget class="QLabel" name="lEnterAmmunition">
     <property name="text">
      <string>Обреіить озброєння :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="saAmmuliton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>402</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lAutomatic">
         <property name="text">
          <string>Автомати</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwAutomatic">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lPistols">
         <property name="text">
          <string>Пістолети </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwPistols">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lGrenadeLaunchers">
         <property name="text">
          <string>Гранатомети</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwGrenadeLaunchers">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Виберіть звання:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbRunk"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="bubOk">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bubCancel">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bubOk</sender>
   <signal>accepted()</signal>
   <receiver>AddNewMilitaryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bubOk</sender>
   <signal>rejected()</signal>
   <receiver>AddNewMilitaryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
